javaparser-core/src/main/javacc/java.jj

```diff
@@ -1693,17 +1693,19 @@ Expression UnaryExpressionNotPlusMinus():
 	JavaToken begin = INVALID;
 }
 {
-  (
-	  ( "~" { op = UnaryExpr.Operator.BITWISE_COMPLEMENT; begin=token(); } |
-	    "!" { op = UnaryExpr.Operator.LOGICAL_COMPLEMENT;     begin=token(); }
-	  ) ret = UnaryExpression() { ret = new UnaryExpr(range(begin, token()), ret, op); }
-	|
-	  LOOKAHEAD( CastExpression() )
-	  ret = CastExpression()
-	|
-      ret = PostfixExpression()
-  )
-  { return ret; }
+	(
+		(
+			"~" { op = UnaryExpr.Operator.BITWISE_COMPLEMENT; begin=token(); } |
+			"!" { op = UnaryExpr.Operator.LOGICAL_COMPLEMENT;     begin=token(); }
+		) ret = UnaryExpression() { ret = new UnaryExpr(range(begin, token()), ret, op); }
+		| LOOKAHEAD( CastExpression() )
+			ret = CastExpression()
+		|
+			ret = PostfixExpression()
+		|
+			ret = SwitchExpression()
```

Comment:
It's not very clear since I reformatted everything, but this choice is everything I added here.